webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsManager.js

```diff
@@ -92,19 +120,21 @@ export class SakaiRubricsManager extends RubricsElement {
         <h3>${this.tr("public_rubrics_title")}</h3>
         <p>${this.tr("public_rubrics_info")}</p>
 
-        <div id="shared-rubrics-title"
-             aria-expanded="${this.sharedRubricsExpanded ? "true" : "false"}"
-             role="tab"
-             aria-multiselectable="true"
-             class="manager-collapse-title"
-             title="${this.tr("toggle_shared_rubrics")}" tabindex="0" @click="${this.toggleSharedRubrics}">
-          <div>
+        <div class="d-flex align-items-center">
+          <button class="btn btn-icon btn-xs me-1"
+              id="shared-rubrics-toggle"
+              data-bs-toggle="collapse"
+              data-bs-target="#shared_rubrics"
+              aria-expanded="false"
+              title="${this.tr("toggle_shared_rubrics")}"
+              aria-label="${this.tr("toggle_shared_rubrics")}"
+              aria-controls="shared_rubrics">
             <span class="collpase-icon fa fa-chevron-right"></span>
-            ${this.tr("shared_rubrics")}
-          </div>
+          </button>
+          <span class="fw-bold">${this.tr("shared_rubrics")}</span>
```

Comment:
This used to be in the button, is that right to display here?

Reply:
I did that on purpose. The button is now just the icon so I moved the title into a span to the right of it.

Reply:
Okay, could you add a screenshot for this as well please?

Reply:
Yep, will do.

Reply:
Now I see, thanks!